docker-compose/common-services/anchor-platform-config.yaml

```diff
@@ -0,0 +1,335 @@
+version: 1
```

Comment:
This file is not necessary. We should go with `env` only configuration for this version.

Reply:
Removed